about.html

```diff
@@ -0,0 +1,111 @@
+          </ul>
+        </nav>
+      </div>
+      <!-- Hamburger icon -->
+      <label id="second-item" class='menu-button-container' for="menu-button-container">
+        <img id="cancel-icon" src="assets/images/close_mobile_menu.png" alt="Cancel Menu">
+        <img id="menu-icon" src="assets/images/mobile_menu.png" alt="Cancel Menu">
+      </label>
+    </div>
+  </header>
+  <!-- Main content -->
+  <main class="content">
+    <section id="about">
+      <h1 id="title">
+        LOME INTERNATIONAL FINTECH CHALLENGE 2022
+      </h1>
+      <p id="description">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Neque gravida in fermentum et sollicitudin ac orci phasellus. Eu sem integer vitae justo eget magna. Id leo in vitae turpis massa sed elementum tempus. Sapien nec sagittis aliquam malesuada bibendum arcu vitae elementum curabitur.
+      </p>
+      <p id="know-more">
+        For any further questions about LOME FINTECH CHALLENGE 2022 please contact us per email 
+        
+      </p>
+      <p id="email">        
+        <a href="mailto:lomefintechchallenge@glft.org" class="contact-email">lomefintechchallenge@glft.org</a>
+      </p>
+    </section>
+    <section id="logo-history">
+      <h1>Lome Fintech Challenge 2022 logo</h1>
+      <span class="logo-history-bar"></span>
+      <p>
+        The logo of Lome Fintech Challenge 2022 was decided through the logo competition from 11. May to 12 June.
+      </p>
+      <img src="assets/images/Lome_Fintech_LogoB_59.png" alt="">
+    </section>
+    <section id="challenge-history">
+      <span class="challenge-history-line"></span>
+      <h1>See the Past Lome  Fintech Challenge</h1>
+      <span class="challenge-history-bar"></span>
+      <p>
+        Take a look at the last two Lome  Fintech Challenge which took place à Kara and Kpalime
+      </p>
+      <div class="container">
+        <img src="assets/images/fellowship.jpg" alt="">
+        <div class="text-block">
+          <h4>2021</h4>
+        </div>
+      </div>
+      <div class="container">
+        <img src="assets/images/fellowship.jpg" alt="">
+        <div class="text-block">
+          <h4>2020</h4>
+        </div>
+      </div>
+    </section>
+    <section id="about-partner">
+      <h1>Partner</h1>
+      <span class="partner-bar"></span>
+      <ul>
+        <li><img src="assets/images/sonsors/iamport.png" alt=""></li>
+        <li><img src="assets/images/sonsors/kakao.png" alt=""></li>
+        <li><img src="assets/images/sonsors/line-friends.png" alt=""></li>
+        <li><img src="assets/images/sonsors/mozilla.png" alt=""></li>
+      </ul>
+
+    </section>
+  </main>
+  <footer id="about-footer">
+   <img src="assets/images/Lome_Fintech_LogoB_59.png" alt="">
+    <p>
+      @2022 Lome Fintech Challenge <br> Some Rights Reserved
+    </p>
+
```

Comment:
Taking a closer look at Cindy's design, you could see that the about page does not contain the partners' section. Also the footer is dark colored as opposed to the home page. It'll be great that you implement this in your design too.

Your design:
![image](https://user-images.githubusercontent.com/20554583/198849382-43384457-3519-430d-88c8-9e7641de9602.png)


Cindy's design:
![image](https://user-images.githubusercontent.com/20554583/198849365-a6b09d3f-7334-439a-a981-08e6053a462f.png)